app/controllers/home_controller.rb

```diff
@@ -1,6 +1,13 @@
 class HomeController < ApplicationController
   helper PostsHelper
   def index
-    @posts = Post.order("created_at").last(3).reverse
+    # Select 3 posts within last 2 days with highest trending score
+    day_limit = 2
+    num_post = 3
+    @posts = Post
```

Comment:
Since this may return < 3 posts, I think we should fill the remaining slots with the most recent posts.

Reply:
Thank you for the reviews! And yes will do.